dev/tools/grunt/tasks/static.js

```diff
@@ -17,7 +17,7 @@ module.exports = function (grunt) {
         };
 
     grunt.registerTask('static', function (target) {
-        var currentTarget = target || 'test',
+        var currentTarget = target || 'file',
```

Comment:
Currently we use `grunt static` to run all test with generating log file and `grunt static:file --file=<path_to_file>` for single file validation. What reason to switch this?